### ISSUE TITLE
nano: ucode syntax highlighting breaks when encountering "}}"

### ISSUE THREAD

**Issue by eric-j-ason** [openwrt/packages#24325]
Maintainer: @hnyman
Environment: master

Syntax highlighting of ucode stops working when the string `}}` is encountered in the source code.

```ucode
#! /usr/bin/env ucode

({a: {}});

// Everyting from here on gets the same color.
print("Hello!\n");
0 + 1 + 2 + 3 + 4 + 5;
return null;
```

<img width="496" alt="bild" src="https://github.com/openwrt/packages/assets/75301977/19c5bf13-185e-4fa3-bd71-42f095249dcf">

**Comment by dave14305** [comment]
It’s working, but the use of a double `}}` triggers the template rule. If you separate them by a space, it should snap back to normal, no?

https://github.com/openwrt/packages/blob/4ef2b7e0fc3d0cb714e7522b8c9fc57185681560/utils/nano/files/ucode.nanorc#L44-L45

**Comment by eric-j-ason** [comment]
> It’s working, but the use of a double `}}` triggers the template rule. If you separate them by a space, it should snap back to normal, no?

Indeed, that's my current workaround, but it's ugly and I don't want to have to do that! :) I'm not using templates at all.
